internal/cli/opsmanager/clusters/delete.go

```diff
@@ -27,10 +29,13 @@ import (
 	"go.mongodb.org/ops-manager/atmcfg"
 )
 
+const defaultWait = 4 * time.Second
```

Comment:
[nit] feel free to use [watch_opts](https://github.com/mongodb/mongocli/blob/master/internal/cli/watch_opts.go) to handle this and  the polling mechanism, it would be nice to add the spinner as this command is going to lock the terminal (a bit of an odd one as we usually don't do that)